service/src/main/resources/db/migration/V95__message_account_foreign_keys.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE message_account DROP CONSTRAINT message_account_created_for_fk;
+
+-- CASCADE also drops the message_account_name_view, but it's recreated in an R__ migration
+ALTER TABLE message_account DROP COLUMN deleted_owner_name CASCADE;
+
+ALTER TABLE message_account ADD CONSTRAINT message_account_created_for_fk CHECK (
+    num_nonnulls(daycare_group_id, employee_id, person_id) = 1
+);
```

Comment:
Doesn't this now have a slight chance of failing if someone has already deleted a message group with messages?
It shouldn't be the case, but let's keep eyes open when deploying.

Reply:
Yes, has to be monitored